vnext/Desktop.IntegrationTests/React.Windows.Desktop.IntegrationTests.vcxproj

```diff
@@ -64,35 +64,30 @@
       <PreprocessorDefinitions>OSS_RN;FOLLY_MOBILE=1;ENUM_BITFIELDS_NOT_SUPPORTED;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
   </ItemDefinitionGroup>
-  <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
-    <ClCompile>
-      <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-    </ClCompile>
-  </ItemDefinitionGroup>
-  <ItemDefinitionGroup Condition="'$(Configuration)'=='Release'">
-    <ClCompile>
-      <Optimization>MaxSpeed</Optimization>
-      <FunctionLevelLinking>true</FunctionLevelLinking>
-      <IntrinsicFunctions>true</IntrinsicFunctions>
-      <PreprocessorDefinitions>NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-    </ClCompile>
-    <Link>
-      <EnableCOMDATFolding>true</EnableCOMDATFolding>
-      <OptimizeReferences>true</OptimizeReferences>
-    </Link>
-  </ItemDefinitionGroup>
   <ItemGroup>
     <ClCompile Include="ChakraJSIRuntimeHolder.cpp" />
     <ClCompile Include="HttpResourceIntegrationTests.cpp" />
-    <ClCompile Include="RNTesterIntegrationTests.cpp" />
-    <ClCompile Include="DesktopTestInstance.cpp" />
+    <ClCompile Include="RNTesterIntegrationTests.cpp">
+      <!-- TODO: Remove NOJSC from react-native. -->
+      <PreprocessorDefinitions>NOJSC;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
>NOJSC [](start = 31, length = 5)

Seeing the NOJSC "literal" in various places makes me wonder how many places would have to be touched if this setting needs to change (and the chances of us missing a spot). Would it it be better to use a centrally managed property instead (i.e. replacing the 'NOJSC' here with '$(OurCentralNoJSCSetting)'?

Reply:
This is part of a two-step effort to get rid of the flag. It is no longer needed.
I have a parallel change in `react-native`, which removes all references to this macro.
Once that dependency is updated, we can fully remove `NOJSC` from this code base.

This change just minimizes/compacts its usage into a few project definitions, but leaving it out of the source code.